docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -1,97 +1,34 @@
-It can also protect hosts from security threats, query data from operating systems, forward data from remote services or hardware, and more.
 
-The {agent} approach works well if you want to configure and update monitors using a UI in your browser.
-While there are {fleet-guide}/migrate-beats-to-agent.html#why-migrate-to-elastic-agent[many benefits to using {agent}], the Elastic Synthetics integration is currently in beta and is subject to change.
+Read more in <<uptime-set-up-choose-heartbeat>>.
+| **{uptime-app}** | beta[] The {uptime-app} is an application in {kib}.
+You can now create monitors directly in the {uptime-app}.
+To try out this beta functionality, you will need to enable Monitor Management.
 
-The {agent} approach may not work for all use cases.
-This method does not support custom location names so is only suitable for running checks from a single location.
-Also, if you want to create and delete monitors programmatically, using {agent} can be less convenient than using <<uptime-set-up-choose-heartbeat>> directly.
+This approach works well if you want to configure and update monitors using a UI in your browser.
+However, **this functionality is in beta and is subject to change.**
 
-If you are using the {agent} approach, there are also two prerequisites to complete before you can configure monitors:
-
-* *Set up {fleet-server}*: If you are using {ecloud}, {fleet-server} will already be provided and you can skip this step.
-To learn more, see <<set-up-fleet,{fleet}>>.
-* *Create a separate {fleet} policy to use for Synthetics monitors*: Generally, synthetics monitors should run on dedicated nodes.
-For more information on {fleet} policies and creating them, see {fleet-guide}/agent-policy.html#create-a-policy[{agent} policy].
-
-[discrete]
-[[uptime-set-up-instructions]]
-== Set up
-
-Once you've chosen an approach, you can begin setting up your first monitor.
-
-[discrete]
-[[uptime-set-up-docker]]
-=== Pull the Docker image
-
-Start by pulling the Docker image.
-
---
-include::tab-widgets/uptime-monitoring/set-up-docker-widget.asciidoc[]
---
-
-[discrete]
-[[uptime-set-up-config]]
-=== Configure
-
-Configure the monitor using either a `heartbeat.yml` configuration file or through the {agent} UI.
-
---
-include::tab-widgets/uptime-monitoring/set-up-conf-widget.asciidoc[]
---
-
-[discrete]
-[[uptime-set-up-connect]]
-=== Connect to the {stack}
-
-After configuring the monitor, run it in Docker and connect the monitor to the {stack}.
-
---
-include::tab-widgets/uptime-monitoring/set-up-connect-widget.asciidoc[]
---
-
-[discrete]
-[[uptime-set-up-kibana]]
-=== View in {kib}
+Read more in <<uptime-set-up-choose-agent>>.
+|===
 
-Elastic Synthetics is now sending synthetic monitoring data to the {stack}.
-Navigate to the {uptime-app} in {kib}, where you can see screenshots of each run,
-set up alerts in case of test failures, and more.
+// Should we add `push` to this table as another option?
```

Comment:
Should we add `push` to this table as another option?

Reply:
Yeah, I think that makes sense

Reply:
I'm going to do this in a follow-up PR to avoid scope creep here.